timemachine/fe/free_energy.py

```diff
@@ -778,7 +778,8 @@ def compute_intermediate_result(lambdas: Sequence[float]) -> PairBarResult:
             lamb1 = lambdas[left_idx]
             lamb2 = lambdas[left_idx + 1]
             print(
-                f"Current minimum BAR overlap {cost_to_overlap(max(costs)):.3g} "
+                f"Bisection iteration {iteration} (of {n_bisections}): "
+                f"Current minimum BAR overlap {cost_to_overlap(max(costs)):.3g} <= {min_overlap:.3g}"
```

Comment:
Need a space in there to avoid having `0.667between`

```suggestion
                f"Current minimum BAR overlap {cost_to_overlap(max(costs)):.3g} <= {min_overlap:.3g} "
```